data/test/scenarios/overlay_text_demo.cfg

```diff
@@ -0,0 +1,270 @@
+#textdomain wesnoth-test
+
+[test]
+    name = _ "Overlay Text Demo"
+    map_file=test/maps/generic_unit_test.map
+    id = overlay_text_demo
+    turns = unlimited
+    
+    [side]
+        side=1
+        controller = human
+        suppress_end_turn_confirmation = yes
+        [leader]
+            name = _ "Alice"
+            type = Elvish Archer
+            id=alice
+            [modifications]
+                {TRAIT_QUICK}
+            [/modifications]
+        [/leader]
+    [/side]
+    [side]
+        side=2
+        controller = human
+        suppress_end_turn_confirmation = yes
+        [leader]
+            name = _ "Bob"
+            type = Orcish Grunt
+            id=bob
+            [modifications]
+                {TRAIT_QUICK}
+            [/modifications]
+        [/leader]
+    [/side]
+    [event]
+        name=preload
+        [lua]
+            code=<<
+                left_label_id = {valid = false}
+                right_label_id = left_label_id
```

Comment:
While copying the values like this works, it would be more readable to just put `{valid = false}` on each line.